public/pages/Main/Main.tsx

```diff
@@ -121,6 +133,9 @@ export default class Main extends Component<MainProps, MainState> {
       selectedNavItemId: 1,
       dateTimeFilter: defaultDateTimeFilter,
       findingFlyout: null,
+      dataSourceLoading: props.multiDataSourceEnabled,
+      selectedDataSource: { id: '' },
```

Comment:
same comment, is the default local cluster?

Reply:
+1

Reply:
Yes, local cluster is default for now. I am still understanding how Neo would function without local cluster and if we need to wait on loading the entire plugin until we get the proper data source